sdk/storage/azdatalake/internal/generated/autorest.md

```diff
@@ -208,11 +208,13 @@ directive:
 
 ``` yaml
 directive:
-  - from: zz_service_client.go
-    where: $
-    transform: >-
-      return $.
-        replace(/req.Raw\(\).URL.RawQuery \= reqQP.Encode\(\)/, `req.Raw().URL.RawQuery = strings.Replace(reqQP.Encode(), "+", "%20", -1)`)
+- from: 
+  - zz_service_client.go
+  - zz_container_client.go
```

Comment:
Shouldn't this change reflect in both zz_service_client.go and zz_container_client.go in datalake's generated code?

Reply:
Since the transform already existed in zz_service_client.go, there is no diff. The change is still there. https://github.com/Azure/azure-sdk-for-go/blob/0de654744232cf8aaed01dee831bb81ea1f97b02/sdk/storage/azdatalake/internal/generated/zz_service_client.go#L52

Reply:
The above line needs to be corrected from `zz_container_client.go` to `zz_filesystem_client.go`. The transformation needs to be applied to filesystem client.
https://github.com/Azure/azure-sdk-for-go/blob/98591cf681ee73baadf93884f201d3ffbefb13d0/sdk/storage/azdatalake/internal/generated/zz_filesystem_client.go#L62